app/scripts/ui.js

```diff
@@ -98,10 +102,21 @@ async function start() {
   const messageListener = async (message) => {
     const method = message?.data?.method;
 
-    if (method !== METHOD_START_UI_SYNC) {
-      return;
+    switch (method) {
+      case METHOD_START_UI_SYNC:
+        await handleStartUISync();
+        break;
+      case METHOD_DISPLAY_STATE_CORRUPTION_ERROR:
+        handleDisplayStateCorruptionError(message.data.params);
+        break;
+      case 'RELOAD':
```

Comment:
I'm not seeing this method referenced anywhere. Was this accidentally included in changes brought in from a future PR?

Reply:
yes, good catch. i've removed it!